frontend/src/employee-frontend/components/reports/AssistanceNeedDecisionsReportDecision.tsx

```diff
@@ -0,0 +1,205 @@
+import { CollapsibleContentArea } from 'lib-components/layout/Container'
+import StickyFooter from 'lib-components/layout/StickyFooter'
+import { FixedSpaceRow } from 'lib-components/layout/flex-helpers'
+import InfoModal from 'lib-components/molecules/modals/InfoModal'
+import { H2 } from 'lib-components/typography'
+import { defaultMargins, Gap } from 'lib-components/white-space'
+import { faQuestion } from 'lib-icons'
+
+import AssistanceNeedDecisionReadOnly from '../child-information/assistance-need/decision/AssistanceNeedDecisionReadOnly'
+import {
+  decideAssistanceNeedDecision,
+  getAssistanceNeedDecision,
+  markAssistanceNeedDecisionAsOpened
+} from '../child-information/assistance-need/decision/api'
+
+import { AssistanceNeedDecisionReportContext } from './AssistanceNeedDecisionReportContext'
+
+const StickyFooterContainer = styled.div`
+  padding: ${defaultMargins.xs};
+`
+
+const DangerAsyncButton = styled(AsyncButton)`
+  background-color: ${(p) => p.theme.colors.status.danger};
+  color: ${(p) => p.theme.colors.grayscale.g0};
+  border-color: transparent;
+
+  &:hover,
+  &:active {
+    background-color: ${(p) => shade(0.1, p.theme.colors.status.danger)};
+    color: ${(p) => p.theme.colors.grayscale.g0};
+    border-color: transparent;
+  }
+`
+
+export default React.memo(function AssistanceNeedDecisionsReportDecision() {
+  const { id } = useNonNullableParams<{ id: UUID }>()
+  const navigate = useNavigate()
+
+  const [assistanceNeedDecision, reloadDecision] = useApiState(
+    () => getAssistanceNeedDecision(id),
+    [id]
+  )
+
+  const { user } = useContext(UserContext)
+  const { refreshAssistanceNeedDecisionCounts } = useContext(
+    AssistanceNeedDecisionReportContext
+  )
+
+  useEffect(() => {
+    if (!user) {
+      return
+    }
+
+    void assistanceNeedDecision.map(async ({ decision }) => {
+      if (decision.decisionMaker?.employeeId === user?.id) {
+        await markAssistanceNeedDecisionAsOpened(decision.id)
+        refreshAssistanceNeedDecisionCounts()
+      }
+    })
+  }, [assistanceNeedDecision, user, refreshAssistanceNeedDecisionCounts])
+
+  const {
+    i18n: {
+      childInformation: { assistanceNeedDecision: t },
+      ...i18n
+    }
+  } = useTranslation()
+
+  const [appealInstructionsOpen, setAppealInstructionsOpen] = useState(false)
+
+  const [returnForEdit, setReturnForEdit] = useState(false)
+
+  return (
+    <>
+      {!!returnForEdit && (
+        <ReturnForEditDecisionModal
+          onClose={(shouldRefresh) => {
+            setReturnForEdit(false)
+            if (shouldRefresh) {
+              reloadDecision()
+            }
+          }}
+          decisionId={id}
+        />
+      )}
+
+      <Content>
+        <ReturnButton label={i18n.common.goBack} />
+
+        <ContentArea opaque>
+          {renderResult(assistanceNeedDecision, ({ decision }) => (
+            <AssistanceNeedDecisionReadOnly decision={decision} />
```

Comment:
This, along with the appeal instructions below, should also be wrapped in an `I18nContext.Provider` using the language of the decision.